src/ray/gcs/gcs_client/accessor.cc

```diff
@@ -459,9 +459,9 @@ void ActorInfoAccessor::AsyncResubscribe() {
   // server first, then fetch data from the GCS server.
   absl::MutexLock lock(&mutex_);
   for (auto &[actor_id, resubscribe_op] : resubscribe_operations_) {
-    RAY_CHECK_OK(resubscribe_op([this, actor_id = actor_id](const Status &status) {
```

Comment:
why is there shadowing in this case?

Reply:
I guess it's the captured `actor_id` shadows the structure binding `actor_id`. The error shows "declaration shadows a structured binding".
![image](https://github.com/user-attachments/assets/3c59e827-f641-44e3-b009-a63107bcc300)